spec/rails_helper.rb

```diff
@@ -9,6 +9,8 @@
 VCR.configure do |config|
   config.cassette_library_dir = "fixtures/vcr_cassettes"
   config.hook_into :webmock
+  # config.configure_rspec_metadata!
+  # config.response.body.force_encoding('UTF-8')
```

Comment:
We should implement the keep api key private before merging this to main and re-record any cassettes

Reply:
thanks it is done